phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -491,6 +491,15 @@ private int rebuildIndexes(String dataTable, IndexTool indexingTool) {
             try {
                 LOGGER.info("Rebuilding index: " + String.join(",", args));
                 if (!dryRun) {
+                    // If the index is in DISABLED state, indexTool will fail. First to ALTER REBUILD ASYNC
```

Comment:
nit: Good for the comment to make clear that ALTER INDEX REBUILD ASYNC will set the index state to BUILDING, which is safe to make ACTIVE later.

Reply:
done